components/brave_wallet/browser/keyring_service.cc

```diff
@@ -683,12 +682,6 @@ HDKeyring* KeyringService::RestoreKeyring(const std::string& keyring_id,
     return nullptr;
   }
 
-  // Some keyrings just create encryptors for lazily keyring creation.
```

Comment:
There were no intent to remove all laziness, only when account is restored since we need all keyrings at this moment. Do you suggest to remove it in this PR?

Reply:
hmm, let's only remove laziness for restore as long as it is mentioned in summary